test/src/player.spec.js

```diff
@@ -1649,3 +1649,58 @@ describe('seekToLiveEdge', function () {
     }
   });
 });
+
+describe('volume', function () {
+  let player, config;
+
+  before(() => {
+    createElement('DIV', targetId);
+  });
+
+  beforeEach(() => {
+    config = getConfigStructure();
+  });
+
+  afterEach(() => {
+    player.destroy();
+  });
+
+  after(() => {
+    removeVideoElementsFromTestPage();
+    removeElement(targetId);
+  });
+
+  it('should return 1 by default', function () {
+    config.sources = sourcesConfig.Mp4;
+    player = new Player(targetId, config);
```

Comment:
lines 1674 and 1675 are repeated in each unit test,
so it's better to place them inside the beforeEach